user-guide/consumers.md

```diff
@@ -80,7 +80,7 @@ SomeConsumer.start('someStream')
 A consumer's `start` method offers a number of parameters to control its timing.
 
 ``` ruby
-self.start(stream_name, poll_interval_milliseconds: 100, batch_size: 1000, position_update_interval: 100, position_store: nil, session: nil)
+self.start(stream_name, poll_interval_milliseconds: 100, batch_size: 1000, position_update_interval: 100, settings: nil, condition: nil)
```

Comment:
This seems like it should be documented as session coming at the end of the list of parameters